src/extension/wps/src/main/java/au/org/emii/utils/GoGoDuckConfig.java

```diff
@@ -110,12 +115,12 @@ public String getLongitudeEnd(String layer) throws Exception {
         return getConfig(LONGITUDE_END, getLayerConfigFilePath(layer));
     }
 
-    public String getTimeStart(String layer) throws Exception {
-        return getConfig(TIME_START, getLayerConfigFilePath(layer));
+    public Map<String, String> getTimeStart(String layer) throws Exception {
+        return getConfigMap(URL_SUBSTITUTION, getLayerConfigFilePath(layer));
```

Comment:
@bhasin - using getConfigMap doesn't work - there are no key attributes - so no time attribute updates are ever made here.  You should use getConfigList instead